molfeat/trans/graph/adj.py

```diff
@@ -736,14 +740,15 @@ def get_collate_fn(
         Get collate function for pyg graphs
 
         Args:
+            dataset: The dataset from which to load the data and apply the collate function.
             follow_batch: Creates assignment batch vectors for each key in the list. (default: :obj:`None`)
             exclude_keys: Will exclude each key in the list. (default: :obj:`None`)
             return_pair: whether to return a pair of X,y or a databatch (default: :obj:`True`)
 
         Returns:
             Collated samples.
         """
-        collator = Collater(follow_batch=follow_batch, exclude_keys=exclude_keys)
+        collator = Collater(dataset=dataset, follow_batch=follow_batch, exclude_keys=exclude_keys)
```

Comment:
Note that `dataset` can be None,  so it's in fact optional to provide (unless your dataset is an `OnDiskDataset` dataset), which the pyg documentation does not mention unfortunately